cmd/phlare/main.go

```diff
@@ -49,14 +53,20 @@ func main() {
 		flags mainFlags
 	)
 
-	if err := cfg.DynamicUnmarshal(&flags, os.Args[1:], flag.CommandLine); err != nil {
+	if err := cfg.DynamicUnmarshal(&flags, os.Args[1:], flag.CommandLine, testMode); err != nil {
```

Comment:
I am not sure does this is a good way. Maybe it is too ugly. If you have any suggestions please feel free to comment. Thanks!

Reply:
https://github.com/grafana/phlare/pull/542/files#r1116866328